src/SAML2/Exception/UnparseableXmlException.php

```diff
@@ -14,7 +14,7 @@ public function __construct(\LibXMLError $error)
     {
         $message = sprintf(
             'Unable to parse XML - "%s[%d]": "%s" in "%s" at line %d on column %d"',
-            static::$levelMap[$error->level],
+            self::$levelMap[$error->level],
```

Comment:
What's the reason for this change? I haven't even looked into the code, but both `self` and `static` are valid keywords with different usages, so I'm a bit puzzled that this change is happening in a code quality revision...

Reply:
Since SAML2\Exception\UnparseableXmlException is declared final, using late-static binding will have no effect.

Reply:
Perfect! Thanks for taking a look into it Tim 😄